unstable/wlr-output-manager-v1.xml

```diff
@@ -0,0 +1,350 @@
+
+        When the client binds to the output_device object, the server sends this
+        event once for every supported mode.
+        
+        Not all output devices need to have supported modes. For example, an
+        RDP-style output might have no advertised modes.
+
+        The size described by a mode represents the size of the underlying buffer.
+        It does not need to be the same as the output size in the compositor space. 
+      </description>
+      <arg name="width" type="int" summary="width of the output buffer"/>
+      <arg name="height" type="int" summary="height of the output buffer"/>
+      <arg name="refresh" type="int" summary="vertical refresh rate in mHz"/>
+    </event>
+    
+    <event name="active_mode">
+      <description summary="active mode of the output">
+        The active_mode event describes the mode that the output device is
+        currently set to.
+
+        The active mode does not need to be one of the modes advertised by
+        the supported_mode event.
+
+        This event is sent after the client binds to the output_device and
+        whenever the mode of the output changes.
+      </description>
+      <arg name="width" type="int" summary="width of the output buffer"/>
+      <arg name="height" type="int" summary="height of the output buffer"/>
+      <arg name="refresh" type="int" summary="vertical refresh rate in mHz"/>
+    </event>
+    
+    <event name="transformation">
+      <description summary="transformations applied to framebuffer">
+        This event describes the transformations that are applied to the
+        framebuffer before it is displayed.
+
+        This event is sent after the client binds to the output_device and
+        whenever the mode of the output changes.
+      </description>
+      <arg name="transformation" type="int" enum="wl_output.transform"/>
+    </event>
+    
+    <event name="size">
+      <description summary="size of the output in the compositor space">
+        The size event describes the size of the output in the global
+        compositor space.
+
+        It is comparable to the zxdg_output_v1.logical_size event, but is not
+        the same in all circumstances. The zxdg_output_v1.logical_size event
+        will not be the same if the compositor does not scale the surface
+        buffers.
+
+        Combined with the current_mode and transformation event, this event
+        effectively describes the scale applied to the output's buffer
+        before it is displayed.
+
+        This event is sent after the client binds to the output_device object
+        and whenever the size of the output device in the global compositor
+        space changes.
+      </description>
+      <arg name="width" type="int"
+        summary="width in global compositor space"/>
+      <arg name="height" type="int"
+        summary="height in global compositor space"/>
+    </event>
+
+    <event name="position">
```

Comment:
I wonder if it makes sense to have these events at all. These events only make sense if the output is enabled. In this case this data can be obtained from wl_output and xdg-output as normal.

The only added value provided by output_device is that the object exists even if the output is disabled. So I wonder if it's necessary to add these at all.

Reply:
The only two events that we would be able to drop are transformation and position. The others are different from the wl_output and xdg_output interfaces. It doesn't feel right to to reference only those two.

Reply:
(Same for "size")

Reply:
(And "active_mode")

Reply:
size is different from the xdg_output logical_size in the current description.

Reply:
hmm, if xdg_output.logical_size would be exactly the same, I'd rather have them separate too. That would solve https://github.com/swaywm/wlr-protocols/pull/23#discussion_r202509810 .